extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/DefaultTenantConfigResolver.java

```diff
@@ -77,13 +78,15 @@ void setTenantConfigContextFactory(Function<OidcTenantConfig, TenantConfigContex
     }
 
     private TenantConfigContext getTenantConfigFromTenantResolver(RoutingContext context) {
+        Assert.assertNotNull(staticTenantsConfig);
```

Comment:
`assertNotNull` will only be triggered if asserts are enabled in the code. 

Maybe `Asserts.checkNotNullParam` is better,  but since `staticTenantsConfig` is not a parameter, I wonder if a simple `if (staticTenantsConfig == null) { throw new IllegalStateException() }` isn't easier :thinking:

Reply:
@gastaldi How does it look now ? (FYI, I've dropped a `dynamicTenantsConfig` map check as it is initialized in a `PostConstruct` method)